app/models/shared/license_module.rb

```diff
@@ -31,6 +31,10 @@ module Shared::LicenseModule
     CC_BY_NC_SA,
     CC0
   ]
+  ODP_LICENSES = [
```

Comment:
Can we get a comment here explaining what ODP means (I assume Open Data Project, but would be nice to leave a note).

Reply:
I added a comment. Feel free to modify or expand on it